### ISSUE TITLE
write report on xview limitations for jatic

### ISSUE THREAD

**Issue by mwartell** [twosixlabs/armory-library#17]
Through extensive work producing and publishing the Honaker/xview_dataset for HuggingFace, followed by later efforts to train a model on that dataset, we have concluded that there are attributes of xView which make it sub-optimal for JATIC purposes.
We will produce a short position paper outlining the limitations and deficiencies that we've uncovered, their historical causes, and a proposal for either:
- the use of a subset of xView, or
- the use of another dataset in its stead, for example euroSat

**Comment by Honaker** [comment]
It seems from our experimentation and usage of the xview dataset that it has many limitations for the jatic purposes. The xview dataset seems to be a former competition dataset for teams to try to create the best model to use object detection to detect objects in the dataset.  This means that there are very few implementations of the xview dataset online. There is only one yolo model that the cdoa management sent out. We were unable to find other implementations online. The issue with the yolo model is that with jatic armory we do not currently have the capability to use yolo models yet. We currently only have torchvision and huggingface models in armory that we can use off the shelf. We will need to spend time adding the implementation to get yolo models in armory. We were able to add a version of the xview dataset to huggingace as a dataset card. We didn’t have a model, so we attempted to train an object detection model on our xview implementation. We however, ran into many issues because the dataset is very complex with many objects in each image and 63 different classes to predict on. It doesn’t seem possible to simply train a model on the dataset. We instead went with the EuroSat dataset for the demo.
We did see the subset of the xview dataset that another cdoa team created with a smaller subset of the dataset with 10 classes to predict on now. We were able to train a simple xview model on that subset of the dataset that produces a nonzero output. We were not able to get a really accurate model however for the same reasons since it is a really complex dataset.
We also ran into the problem of having the xview dataset getting DMCA on huggingface. This also caused problems which forced us to add the dataset to an s3 bucket to be able to use the dataset in armory-library.
